packages/itinerary-body/package.json

```diff
@@ -10,9 +10,9 @@
   "license": "MIT",
   "private": false,
   "dependencies": {
-    "@opentripplanner/core-utils": "^11.0.2",
+    "@opentripplanner/core-utils": "^11.1.3",
     "@opentripplanner/humanize-distance": "^1.2.0",
-    "@opentripplanner/icons": "^2.0.5",
+    "@opentripplanner/icons": "^2.0.6",
```

Comment:
Can we please do a merge-in of master and then another update-internal-dependencies!